lkql_checker/share/lkql/access_to_local_objects.lkql

```diff
@@ -0,0 +1,15 @@
+fun local_scope(e) =
+    match e
```

Comment:
I think this is still missing several cases of local variables such as declare blocks (which can also occur in package body elaboration), protected entries. Does SubpBody include protected subprogram bodies? If not, these need to be also included.

Reply:
Is it possible to have declare blocks that are not enclosed in a subprogram or any such construct? Anyway I added them as type of local scope, this seems like a practical function to have laying around.

Reply:
Yes, for package body elaboration code as I mentioned:
package body xx is
...
begin
   <put any Ada code here, including a declare block>
end xx;